web-common/src/features/welcome/WelcomePageRedirect.svelte

```diff
@@ -51,11 +50,11 @@
     }
 
     // DuckDB-backed projects should head to the Welcome page for user-guided initialization
-    if (olapConnector === "duckdb") {
-      ready = true;
-      await goto("/welcome");
-      return;
-    }
+    // if (olapConnector === "duckdb") {
+    //   ready = true;
+    //   await goto("/welcome");
+    //   return;
+    // }
```

Comment:
Committed it unintentionally. Commented it out because a page refresh would redirect we to the welcome page all the time and not let me view the dashboard